deploy/infra/aws/terraform/cloud-native/cluster-infra/input.tf

```diff
@@ -233,3 +233,51 @@ variable "rds_db_port" {
   default     = 5432
   description = "The port on which the RDS instance accepts connections"
 }
+
+###################################################################
+#               Cloud-pipeline script deployment
+###################################################################
+
+variable "deployment_id" {
+  type        = string
+  description = "Deployment Name"
+}
+
+variable "elb_public_subnet" {
```

Comment:
It's not really a public subnet, but just a subnet for ELB (it could be deployed in private sb too)